bindings/protocols/coap/index.html

```diff
@@ -17,6 +17,16 @@
             edDraftURI: "https://w3c.github.io/wot-binding-templates/bindings/protocols/coap/",
             shortName: "wot-coap-binding",
             github: "https://github.com/w3c/wot-binding-templates/tree/main/bindings/protocols/coap",
+            otherLinks: [{
+                key: "Ontology",
+                data: [
+                    {
+                    value: "CoAP in RDF",
+                    href: "../../../ontology/coap.html"
```

Comment:
The CoAP Binding doesn't use this ontology, so I'm not sure if this link belongs here.

Reply:
Do you have the link to the reference ontology file? or do you mean we should update `ontology/coap.html`?

Reply:
`ontology/coap.html` contains the "CoAP in RDF" ontology, which provides vocabulary to record a CoAP message that has been exchanged between a client and a server (cf. [HTTP in RDF](https://www.w3.org/TR/HTTP-in-RDF10/#introduction)). It was agreed that it would be useful to have this as such.

`bindings/protocols/coap/index.html` provides vocabulary to instruct a WoT client how to construct a CoAP request. This vocabulary is separate from the "CoAP in RDF" ontology and is currently only documented in `bindings/protocols/coap/index.html`.

Reply:
I think we should remove the link to it for now but the bigger question is what to do with that other vocabulary. It is still useful for others but not to us. It would be a waste to delete so we can better explain the existence of it in the CoAP Binding and why we are not using it anymore.